src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -254,6 +254,10 @@ class MapdlGrpc(MapdlBase):
         Print the command ``/COM`` arguments to the standard output.
         The default is ``False``.
 
+    disable_run_at_connect: bool, optional
+        Disable the house-keeping commands when connecting.
+        The default is ``False``
+
     channel : grpc.Channel, optional
         gRPC channel to use for the connection. Can be used as an
         alternative to the ``ip`` and ``port`` parameters.
```

Comment:
```suggestion
        gRPC channel to use for the connection. This parameter can be
        used as an alternative to the ``ip`` and ``port`` parameters.
```
While the brief description is a sentence fragment, all content following it should be in clear, complete sentences that include ending punctuation (periods).